spark3/src/main/java/org/apache/iceberg/spark/source/SparkBatchScan.java

```diff
@@ -373,12 +372,5 @@ private Schema expectedSchema() {
       }
       return expectedSchema;
     }
-
-    private NameMapping nameMapping() {
-      if (nameMapping == null) {
-        this.nameMapping = NameMappingParser.fromJson(nameMappingString);
-      }
-      return nameMapping;
-    }
```

Comment:
I think there might be a PR open related to this. However, I don't see this particular function used: https://github.com/apache/iceberg/pull/1399/files

Reply:
I think we can remove this for now.